src/components/common/UserProfile/UserMeta.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { defineMessages } from 'react-intl';
+import { useMediaQuery } from 'react-responsive';
+
+import CopyableAddress from '~shared/CopyableAddress';
+import ExternalLink from '~shared/ExternalLink';
+import Heading from '~shared/Heading';
+import Icon from '~shared/Icon';
+import Link from '~shared/Link';
+import UserMention from '~shared/UserMention';
+import UserAvatar from '~shared/UserAvatar';
+
+import { stripProtocol } from '~utils/strings';
+import { useAppContext } from '~hooks';
+import { User } from '~types';
+
+import query from '~styles/queries.css';
+import styles from './UserMeta.css';
+
+const componentDisplayName = 'common.UserProfile.UserMeta';
+
+const MSG = defineMessages({
+  editProfileTitle: {
+    id: `${componentDisplayName}.editProfileTitle`,
+    defaultMessage: 'Edit Profile',
+  },
+});
+
+interface Props {
+  user: User;
+}
+
+const UserMeta = ({ user: { walletAddress, profile }, user }: Props) => {
+  const { displayName, bio, website, location } = profile || {};
+  const { user: currentUser } = useAppContext();
+
+  const currentWalletAddress = currentUser?.walletAddress;
+
+  const isMobile = useMediaQuery({ query: query.query700 });
```

Comment:
You could change it to @Julianahlk's `useMobile` hook when you rebase: https://github.com/JoinColony/colonyCDapp/pull/100/files#diff-e08c126439ed0646aed221a78f7a2e69fffda573abfde3d671292e790dcb1248